cmd/apprepository-controller/server/controller.go

```diff
@@ -261,18 +262,18 @@ func (c *Controller) processNextWorkItem() bool {
 			// Forget here else we'd go into a loop of attempting to
 			// process a work item that is invalid.
 			c.workqueue.Forget(obj)
-			runtime.HandleError(fmt.Errorf("expected string in workqueue but got %#v", obj))
+			runtime.HandleError(fmt.Errorf("Expected string in workqueue but got %#v", obj))
```

Comment:
Any reason for capitalizing these? I know our codebase is not consistent, but if we were making it consistent, I'd go the other way. See https://google.github.io/styleguide/go/decisions.html#error-strings

Reply:
No reason apart from internal consistency... having mixed-up capitalized log traces was killing me... so I just normalized them using the most common choice (in this piece of code). But I agree, log traces should be lower-cased instead.
If we start changing little by little, at some point we might end up with a consistent codebase :P, which will be helpful if we ever think of exporting metrics with OpenTelemetry or any other stack.

For the record, the regex `(debug|warn|Error|Info)(f?)\(("|')\p{Lu}` yields 549 upper-cased occurrences across 66 `.go` files. For now, I'm going to lower-case the ones in apprepo controller.

Reply:
Mmm, you're right... I was mixing up the error strings (fmt.Error....) and the log error traces (log.Errorf...), so:

- log.* --> uppercased
- fmt.Errorf --> lowercased

Reverting changes in https://github.com/vmware-tanzu/kubeapps/pull/6647/commits/1696b831364b342887b03c582a045e30c0a9c334 then  :P